docs/usage.rst

```diff
@@ -2,6 +2,12 @@
 Usage
 =====
 
-To use EatFirst fs + flask wrapper in a project::
+To use EatFirst fs + flask wrapper in a project:
 
-	import efs
+.. code-block:: python
+
+    import efs
+
+    # There is no need to initialise it because it will always the current app.
```

Comment:
💄 A verb is missing in the sentence, right? "It will always [?] the current app"